unmarshal.go

```diff
@@ -173,6 +164,9 @@ func findField(fields []FieldData, fieldnum protowire.Number) int {
 }
 
 func tryDecodeFunc(vb []byte, dst reflect.Value) (bool, error) {
+	d := dst.Type().String()
+	_ = d
```

Comment:
i'm lost here :sweat_smile:

Reply:
Ah! This is for debugging using delve - you can't declare variable in Go and not use it, so you have to resort to such things. But I forgot to remove it, even tho I removed all other instances. Thanks, fixed!